packages/nimble-components/.storybook/transformSource.js

```diff
@@ -78,11 +79,8 @@ export const transformSource = source => {
 
     const trimmedHTML = removeBlankLines(html);
     const emptyAttributesRemovedHTML = removeEmptyAttributes(trimmedHTML);
-    const formmattedHTML = prettier.format(emptyAttributesRemovedHTML, {
-        parser: 'html',
-        plugins: [parserHTML],
-        htmlWhitespaceSensitivity: 'ignore',
-        tabWidth: 4
+    const formmattedHTML = beautifyHTML(emptyAttributesRemovedHTML, {
+        wrap_attributes: 'force-expand-multiline'
```

Comment:
FYI @fredvisser updated to this setting so attributes align better to prettier (and pprevents as much horizontal scrolling):

See without this setting (notice need horizontal scroll):
![image](https://github.com/ni/nimble/assets/1588923/af6d80f1-347b-4531-9f80-769f358cd1b0)

With this setting (in their playground, waiting for a build):
![image](https://github.com/ni/nimble/assets/1588923/e8a58350-c4d1-405f-b42c-fa4006771e72)

Previous prettier behavior:
![image](https://github.com/ni/nimble/assets/1588923/41092886-12a8-46ca-bab2-53bb278e06e7)